corehq/pillows/mappings/case_search_mapping.py

```diff
@@ -4,8 +4,8 @@
 from pillowtop.es_utils import ElasticsearchIndexInfo, CASE_SEARCH_HQ_INDEX_NAME
 
 
-CASE_SEARCH_INDEX = prefix_for_tests("case_search_2018-05-29")
-CASE_SEARCH_ALIAS = prefix_for_tests("case_search")
+CASE_SEARCH_INDEX = prefix_for_tests("case_search_2021-06-01")
+CASE_SEARCH_ALIAS = prefix_for_tests("case_search-1")
```

Comment:
It looks like this (trivally) breaks the build. Is the idea here to make a dummy alias that we then switch over to? Is there any reason we can't keep the original alias name and just not flip the alias until the end? I feel like there's something you thought of here that I'm missing

Reply:
The way the code is currently written we assign the alias when the index is created. I think we can revert the alias change once we're ready to roll out. Alternatively I would need need to modify the code to make assigning the alias optional.